docs/commands/artists.md

```diff
@@ -88,14 +88,16 @@ Options:
 
 ### .artisttracks (`.at`)
 
-Shows you your all-time top tracks for the artist you're currently listening to or searching for.
+Shows you your all-time top tracks for the artist you're currently listening to or searching for. You can also show top tracks for someone else in the server.
```

Comment:
Can you make this in line with how we word this in other commands?

So `Shows you or someone else their all-time top tracks for the artist you're currently listening to or searching for.`

And then a list of possible parameters, similarly to the `.topartists` command.